_rules/video-only-transcript-ee13b5.md

```diff
@@ -36,7 +43,7 @@ This rule applies to any [non-streaming](#non-streaming-media-element) `video` e
 
 ## Expectation
 
-The visual information of each test target is available through a text transcript that is [visible][], [included in the accessibility tree][], and is either on the page or linked.
+The visual information of each test target is available through a text transcript that is [visible][], and is either on the page or in a clearly labeled location.
```

Comment:
Same remarks.

For a transcript describing the visual content, the exclusion from the accessibility tree would also be problematic for screen reader users that have then no way to know what is the visual information in the video.